src/pages/RightPane.jsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+
+import SearchBar from "../components/RightPane/SearchBar";
+import Article from "../components/RightPane/Article";
+
+const DUMMY_RESULTS = [
```

Comment:
I'd probably move this to another file somewhere to keep the component uncluttered